lib/settings/pages/settings_page.dart

```diff
@@ -53,9 +55,13 @@ class SettingsPage extends StatelessWidget {
               padding: const EdgeInsets.symmetric(horizontal: 18),
               child: FocusableActionDetector(
                 onFocusChange: (focused) {
-                  if (focused) FocusScope.of(context).unfocus();
+                  if (focused) {
+                    FocusScope.of(context).unfocus();
+                    _searchController.text = '';
```

Comment:
This doesn't clear the text ? [settings_page.dart#L111](https://github.com/thunder-app/thunder/blob/3e7df29bc71e78f8ec2734a8f12676252b31843d/lib/settings/pages/settings_page.dart#L111)

Reply:
Not for me. 😞 Here is the behavior I see in `develop`.

https://github.com/thunder-app/thunder/assets/7417301/227efebd-48fb-4ce0-8c57-e0a3dc60e9a1